gallery/src/components/NxDropdown/NxDropdownPage.tsx

```diff
@@ -85,6 +85,11 @@ const NxDropdownPage = () =>
               <NxTable.Cell>
                 What type of button to render for the dropdown.
                 Defaults to <NxCode>"tertiary"</NxCode>
+
+                <NxWarningAlert>
+                  Deprecated: This prop should not be used.
+                  It produces button styling that does not look good.
```

Comment:
```suggestion
                  It produces button styling that does not comply with design standards.
```

Reply:
Let me know your thoughts on this. Might be a bit better than "does not look good".

Reply:
>Let me know your thoughts on this. Might be a bit better than "does not look good".
sounds good to me!